setup.py

```diff
@@ -20,7 +20,10 @@
     install_requires=[
         'requests',
         'deprecation',
-        'cachetools'
+        'cachetools',
+        'click',
+        'boto3',
```

Comment:
Is boto3 required?

Reply:
https://github.com/intuit/mlctl/blob/main/mlctl/plugins/sagemaker/SagemakerTraining.py

this file needs it.

Reply:
Thanks for pointing that out. 

Two comments: 
1)) We should add more testing so that this would been caught. The library shouldn't have worked for anyone before adding this in. 
2) Do we want to preload all the libraries for each provider? While, We can do the install for boto3 and Google Cloud through PyPI package. However, Azure requires using a [CLI SDK](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli). The installers aren't available on PyPi.